packages/ComponentLibrary/src/components/Table/styles.ts

```diff
@@ -0,0 +1,71 @@
+import { SxProps, Theme } from '@mui/material';
+import { theme } from '../../theme';
+
+export const tableStyles: Record<string, SxProps<Theme>> = {
+  container: {
+    display: 'flex',
+    width: '100%',
+    padding: '1rem 0 1rem 0',
+    flexDirection: 'column',
+    gap: '1rem',
+  },
+  topToolbar: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  tablePaper: {
+    width: '50%',
+    borderRadius: '5rem',
+  },
+  fullScreenContainer: {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    zIndex: 9999,
+    padding: '1rem',
+    gap: '1rem',
+    backgroundColor: 'white',
```

Comment:
maybe you could use the colors provided by the theme

Reply:
This will be fixed in the branch feature/EPL-1704